### DISCUSSION TITLE
AS3 - StageWebView - Error #2044: Unhandled error:. text=PDFError

### DISCUSSION THREAD

**Discussion by zabdoul60460** [airsdk/Adobe-Runtime-Support#1899] [General]
Hello,
I try to display a pdf with a StageWebView component :

    var SWV:StageWebView = new StageWebView();
			var swvHeight:Number = container.height-75
			var swvWidth:Number = container.width;
			var swvYCoord:Number = 75;
			SWV.stage = container.stage;
			SWV.viewPort = new Rectangle(0, swvYCoord, swvWidth, swvHeight);
			var urlToLoad:String = urlCourTXTServ;
			SWV.loadURL(urlToLoad);

For sample, urlToLoad  is like that : 'http://127.0.0.1:10088/MyApp/Data/Customer/File/Demo/w4.pdf'

The path is good, the pdf file exist and I can load it with navigateTourl.

But when I execute the code, this error appear **Error #2044: Unhandled error:. text=PDFError**

Where I'm doing wrong?

Thanks for helping.

**Comment by 2jfw** [comment]
There is kind-of bug with the Webview, when a load operation is called before the previous one did finish - see here: https://github.com/airsdk/Adobe-Runtime-Support/issues/1641

A workaround is to add an ErrorEvent Listener to your webview like

	SWV.addEventListener(ErrorEvent.ERROR,
			                          onError);
